examples/gnrc_minimal/Makefile

```diff
@@ -13,7 +13,7 @@ BOARD_INSUFFICIENT_MEMORY := chronos msb-430 msb-430h
 #USEMODULE += xbee
 
 ## set default UART to use in case none was defined
-#XBEE_UART ?= "UART_NUMOF-1"
+#XBEE_UART ?= "1"
```

Comment:
I don't know what's wrong or not, I just want to have a clear definition of which UART is being used by the XBee device, for me it's somehow ambiguous this `UART_NUMOF-1`, since anyways I'm quite sure to which UART I'm connecting my device.

For the assertion is in this [line](https://github.com/RIOT-OS/RIOT/blob/master/drivers/xbee/xbee.c#L475). Just checks if the UART_NUMOF is greater than the UART device used by the XBee.